python/eos/plot/plotter.py

```diff
@@ -989,6 +989,10 @@ def plot(self):
                 yvalues = np.array(yvalues)
                 yerrors = np.array(yerrors)
 
+                if len(xvalues) == 0:
```

Comment:
Hmm... Not very clean, but I guess we will improve all of this in the new plotting framework :)